clients/src/main/resources/common/message/DescribeQuorumResponse.json

```diff
@@ -38,7 +38,9 @@
           "about": "The latest known leader epoch"},
         { "name": "HighWatermark", "type": "int64", "versions": "0+"},
         { "name": "CurrentVoters", "type": "[]ReplicaState", "versions": "0+" },
-        { "name": "Observers", "type": "[]ReplicaState", "versions": "0+" }
+        { "name": "Observers", "type": "[]ReplicaState", "versions": "0+" },
+        { "name": "HighWatermarkUpdateTimeMs", "type": "int64", "default": -1, "tag": 0,
+          "taggedVersions": "1+", "ignorable": true, "about": "The wall clock time that the high watermark was last updated on the leader" }
```

Comment:
This comment applies to all of the places where you added the `Ms` suffix.

Should it be `HighWatermarkUpdateTimestamp`? I was under the impression that in Kafka the suffix `Timestamp` means wall clock time while the suffix `Ms` means a duration or time span.

We should document that `-1` means that the high watermark update timestamp is not know.